### DISCUSSION TITLE
Assegnazione alunni classe

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#122] [Idee e proposte]
Buonasera,sarebbe possibile facilitare e velocizzare l'assegnazione dell'alunno alla classe?Intendo dire che oltre alla classe che è possibile inserire nel file csv , sarebbe possibile creare un quadratino  da spuntare nominata "seleziona" che selezioni il numero desiderato di alunni e una freccia che indichi direttamente la classe in cui colocare gli alunni con la spunta senza fare alunno per alunno?

**Comment by trinko** [comment]
Non è previsto al momento.
Lo segno come richiesta.
